graveler/ref/branch_iterator_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/treeverse/lakefs/testutil"
 )
 
-func TestPGBranchIterator(t *testing.T) {
+func TestBranchIterator(t *testing.T) {
 	r, db := testRefManagerWithDB(t)
 	branches := []graveler.BranchID{"a", "aa", "b", "c", "e", "d"}
```

Comment:
6 divisible by 3; I think this is unchanged from the previous review where it means that we end up not testing partial fetches (we should do both).  Or, perhaps run the "listing all branches" test in a loop for _all_ batch sizes 1..7?

Reply:
There are 6 branches here + master = so we have 7. I'll make sure tag and repo will have the same.

Reply:
Ouch!  A rich corporation once said, "[math is hard](https://whyplay.mattel.com/the-barbie-vlog-empowers-girls-with-words/)" -- and they're still not sorry!